beesdoo_print_label/views/report_pricetag_normal.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="css_qweb_template_pricetag_normal" inherit_id="web.report_assets_common">
+        <xpath expr="." position="inside">
+            <link href="/beesdoo_print_label/static/css/pricetag_base.scss" rel="stylesheet" type="text/scss"/>
+            <link href="/beesdoo_print_label/static/css/pricetag_normal_base.scss" rel="stylesheet" type="text/scss"/>
+            <link href="/beesdoo_print_label/static/css/pricetag_normal_normal.scss" rel="stylesheet" type="text/scss"/>
+        </xpath>
+    </template>
+
+    <template id="qweb_template_pricetag_normal">
+        <t t-call="web.basic_layout">
+            <t t-foreach="category_data['lines']" t-as="line">
+                <t t-foreach="line.quantity" t-as="q">
```

Comment:
`q` is not used in later, did you actually mean to loop on a quantity ?

Reply:
line.quantity is the number of labels to print for a product.

Reply:
Ok, I did not know you could do a foreach on an integer. 👌